pkgs/applications/science/electronics/fparser/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fparser";
+  version = "unstable-2015-09-25";
+
+  src = fetchFromGitHub {
+    owner = "thliebig";
+    repo = "fparser";
+    rev = "a59e1f51e32096bfe2a0a2640d5dffc7ae6ba37b";
+    sha256 = "0wayml1mlyi922gp6am3fsidhzsilziksdn5kbnpcln01h8555ad";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  enableParallelBuilding = true;
```

Comment:
This becomes redundant once you add `cmake`. I'd link the section in the manual, but the website seems to be down atm... :disappointed:

Reply:
correct, the cmake shell hook will set this to true by default

Reply:
Good catch, I've removed that line.